x-pack/agent/pkg/agent/program/testdata/single_config.yml

```diff
@@ -1,19 +1,49 @@
-streams:
-  - type: metric/docker
-    setting: one
-  - type: metric/apache
-    setting: two
-  - type: event/file
-    paths:
-      - /var/log/hello1.log
-      - /var/log/hello2.log
-management:
-  host: "localhost"
-config:
-  reload: 123
+name:	Production Website DB Servers
+fleet:
+    kibana_url:		https://kibana.mydomain.com:5601
+    ca_hash:		7HIpactkIAq2Y49orFOOQKurWxmmSFZhBCoQYcRhJ3Y=
+    checkin_interval:	5m
```

Comment:
Something weird with the same in a few place, I think YAML will strip it?

Reply:
this is weird i have a single space here